### ISSUE TITLE
Remove superfluous name from tooltip

### ISSUE THREAD

**Issue by deadmeu** [Asthereon/green-thumb#2]
Love the plugin, though I currently only use it to display a tooltip with payment info. Currently, the item's name is rendered first on its own line, which I think is unnecessary. Removing it would simplify the tooltip and reduce clutter when hovering over items in the inventory.

I've had some trouble getting a development environment setup for RuneLite so I can't test this myself but I believe all that needs to change is to remove the string parameter passed into the constructor of this `StringBuilder` object:
https://github.com/Asthereon/green-thumb/blob/dfa65a978e4167df7b185e5b67b300144f4768e7/src/main/java/com/asthereon/green_thumb/Seed.java#L171

**Comment by Asthereon** [comment]
Absolutely agree with this suggestion, working on this next.

**Comment by Asthereon** [comment]
This issue is resolved, just waiting on the update to be approved by RuneLite.

**Comment by deadmeu** [comment]
Super quick turnaround - thanks!

**Comment by Asthereon** [comment]
This has been merged into RuneLite and should now be available, enjoy!
